### PR TITLE
some fixes

### PR DESCRIPTION
**Pull request by yapei**

### CODE REVIEW

File: lib/rules/web/admin_console/4.1/status.xyaml [hunk 1]
```diff
@@ -27,14 +27,8 @@ check_dim_resource_list:
       xpath: //div[@class='co-disabled']//button[contains(.,'Create')]
   - selector:
       xpath: //div[@class='co-disabled']//div[contains(.,'No <resource_singular>s Found')]
-check_user_starter_guide_message_when_no_projects:
```
Comment (yapei) [inline lib/rules/web/admin_console/4.1/status.xyaml]: remove text checking

File: lib/rules/web/admin_console/4.2/status.xyaml [hunk 2]
```diff
@@ -25,14 +25,12 @@ check_dim_resource_list:
...
       xpath: //div[@class='co-disabled']//div[contains(.,'No <resource_singular>s Found')]
-check_user_starter_guide_message_when_no_projects:
-  elements:
-  - selector:
-      visible_text: !ruby/regexp /Create a project/i
 check_get_started_message_when_no_resources:
   elements:
   - selector:
-      visible_text: !ruby/regexp /No Workloads Found/i
+      xpath: //a[.='Import YAML']
```
Comment (yapei) [inline lib/rules/web/admin_console/4.2/status.xyaml]: add link checking

**Comment by yapei** [comment]
@yanpzhan  @SSshahan   could you help review the fix?  Thanks

**Comment by yanpzhan** [comment]
lgtm now.

**Comment by yapei** [comment]
@akostadinov  @pruan-rht  Can you help merge? Thanks

**Pull request merged**